roles/listserv/tasks/main.yml

```diff
@@ -11,8 +11,8 @@
 - name: Update postfix config
   ansible.builtin.lineinfile:
     path: /etc/postfix/main.cf
-    regexp: '^#myhostname = host.domain.tld'
-    line: "myhostname = {{ listserv_api_email_from }}"
+    search_string: '#myhostname = virtual.domain.tld'
```

Comment:
```suggestion
    regexp: '^#?myhostname = '
```

Reply:
The suggestion above will match the myhostname line both when it is commented out as well as when it is active and ignore the current value for myhostname. Hence it will allow changing/updating the value. The `search_string: '#myhostname = virtual.domain.tld'` would only work the first time.